client/settings/index.js

```diff
@@ -1,3 +1,4 @@
+/* global wc_stripe_settings_params */
```

Comment:
Good catch fixing the linting error too 👍 
This LGTM unless you want to add another method to `WC_Stripe_Feature_Flags` here.

Reply:
I'll open a separate PR to include a refactor - I think that the `is_upe_enabled` could be confusing, I'll try and align with the WCPay repo naming 👍